vital/bindings/python/vital/types/detected_object.cxx

```diff
@@ -37,15 +37,31 @@ namespace py = pybind11;
 
 typedef kwiver::vital::detected_object det_obj;
 
+// We want to be able to add a mask in the python constructor
+// so we need a pass-through cstor
+std::shared_ptr<det_obj>
+new_detected_object(kwiver::vital::bounding_box<double> bbox,
+                    double conf,
+                    kwiver::vital::detected_object_type_sptr type,
+                    kwiver::vital::image_container_sptr mask)
+{
+  std::shared_ptr<det_obj> new_obj(new det_obj(bbox, conf, type));
```

Comment:
Would std::make_shared<>() be better here?

Reply:
no significant preference, I'm not making the change because it increases code size by a little unless pushback